tensorflow_datasets/image/the300w_lp.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""300W-LP Dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+_DATASET_URL = "https://drive.google.com/uc?export=download&" \
+               "id=0B7OEHD3T4eCkVGs0TkhUWFN6N1k"
+
+_PROJECT_URL = "http://www.cbsr.ia.ac.cn/users/xiangyuzhu/" \
```

Comment:
hmt'L' missing

Reply:
> hmt'L' missing

the project site is really ended without 'L'.
"http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/main.htm"